requirements/ingest/chroma.in

```diff
@@ -4,3 +4,7 @@ chromadb
 importlib-metadata>=7.1.0
 # Future releases adds in typer-cli which breaks the resolution of typer as a library
 typer<=0.9.0
+# tenacity 9.0.0 is being installed via chroma, but other dependencies (langchain) restrict tenacity
+# to <=9.0.0 in other depenceny extras and resolve to 8.5.0.
+# The original langchain pin: https://github.com/langchain-ai/langchain/pull/849/
+tenacity==8.5.0
```

Comment:
add tenacity pin and explanation for pin in case Langchain updates their pin and we can remove this